smoke_tests/smoke_tests.py

```diff
@@ -0,0 +1,92 @@
+import argparse
+import sys
+import unittest
+from unittest import TestSuite
+import requests
+
+from tests.smoke_test_case import SmokeTestCase
+from tests.unauthenticated.status_tests import StatusTests
+from tests.authenticated.azure.managed_apps_tests import ManagedAppsTests
+from tests.authenticated.billing_profiles import BillingProfileTests
+
+DESCRIPTION = """
+BPM Smoke Test
+Enter the host (domain and optional port) of the BPM instance you want to to test. This test will ensure that the BPM 
+instance running on that host is minimally functional.
+"""
+
+
+def gather_tests(is_authenticated: bool = False, azure_subscription: bool = False) -> TestSuite:
```

Comment:
And actually you are passing the auth token for is_authenticated, aren't you?

Probably fine to leave them as booleans, but make the actual values passed to them booleans.

Reply:
I was just relying on python's type coercion. If it's there it will be truthy, if it's not, it will be falsey.

 I don't mind cleaning it up to be more type-consistent.

Reply:
If you are going to specify the types as booleans, then make them booleans. If you just want to leave off the types then I'm also OK with that… it just gets confusing to me when you say it's a type that it is not.